src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -108,7 +113,8 @@ public AuthenticationResult(
           IEnumerable<string> scopes,
           Guid correlationId,
           AuthenticationResultMetadata authenticationResultMetadata,
-          string tokenType = "Bearer") :
+          string tokenType,
+          X509Certificate2 bindingCertificate) :
```

Comment:
no need to change this ctor

Reply:
without this change, we get this error @bgavrilMS 

`Symbol 'AuthenticationResult' violates the backcompat requirement: 'Do not add multiple overloads with optional parameters'. See 'https://github.com/dotnet/roslyn/blob/main/docs/Adding%20Optional%20Parameters%20in%20Public%20API.md' for details.`